### PR TITLE
Feature/d3 asim 3669: Rebranding

### PR DESCRIPTION
**Pull request by faizan2590**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,22 +27,22 @@
...
 In the following commands for the connection via the REST API are marked with `REST`. 
 
 ## Installation Instructions
 
-Installation of d3a-api-client using pip:
+Installation of gsy-e-sdk using pip:
 
 ```
-pip install git+https://github.com/gridsingularity/d3a-api-client.git
+pip install git+https://github.com/faizan2590/gsy-e-sdk.git
```
Comment (faizan2590) [inline README.md:45]: Reminder to revert to gsy's repo after merge

File: setup.py [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 
 try:
     with open("requirements/base.txt", encoding="utf-8") as req:
         REQUIREMENTS = [r.partition("#")[0] for r in req if not r.startswith("-e")]
         REQUIREMENTS.extend(
-            ["d3a-interface @ "
-             f"git+https://github.com/gridsingularity/d3a-interface.git@{BRANCH}"])
+            ["gsy-framework @ "
+             f"git+https://github.com/faizan2590/gsy-framework@{BRANCH}"])
```
Comment (faizan2590) [inline setup.py:16]: reminder to revert to gsy sdk after merge

File: setup.py [hunk 3]
```diff
@@ -26,20 +26,20 @@
...
 setup(
-    name="d3a-api-client",
-    description="D3A API Client",
+    name="gsy-e-sdk",
+    description="GSy Exchange Software Development Kit",
     long_description=README,
     author="GridSingularity",
     author_email="d3a@gridsingularity.com",
-    url="https://github.com/gridsingularity/d3a-api-client",
+    url="https://github.com/faizan2590/gsy-e-sdk",
```
Comment (faizan2590) [inline setup.py:34]: reminder to revert to gsy sdk after merge

**Review by spyrostz — APPROVED** [review]
LGTM

File: d3a_api_client/__init__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+import gsy_e_sdk
+
+logger = logging.getLogger(__name__)
+console_handler = logging.StreamHandler()
+console_handler.setFormatter(logging.Formatter("%(levelname)s: %(message)s"))
+logger.addHandler(console_handler)
+
+logger.warning(
+    "The d3a_api_client module name will be deprecated soon. Please use gsy_e_sdk instead.")
```
Comment (spyrostz) [inline d3a_api_client/__init__.py:12]: Minor, but please use `The d3a_api_client package name will be deprecated soon.`

**Review by mclrch — DISMISSED** [review]
LGTM

**Review by spyrostz — APPROVED** [review]
LGTM

**Pull request merged**